templates/tasks/scheduled_tasks/new_email_brandings.html

```diff
@@ -1,14 +1,87 @@
-<p>There are new email brands to review since {{ yesterday }}.</p><hr>
+{% if brands_by_organisation %}
+<h2>New email branding to review</h2>
+<p>Uploaded since {{ yesterday }}:</p>
+{% endif %}
 {%- for org, brands in brands_by_organisation.items() -%}
-    <p><a href="{{ domain }}/organisations/{{ org.id }}">{{ org.name }}</a>:</p><ul>
+    <p><a href="{{ domain }}/organisations/{{ org.id }}/settings/email-branding">{{ org.name }}</a>{% if not org.email_branding_id %} (no default){% endif %}:</p><ul>
     {%- for brand in brands -%}
         <li><a href="{{ domain }}/email-branding/{{ brand.id }}/edit">{{ brand.name }}</a></li>
     {%- endfor -%}
     </ul><hr>
 {%- endfor -%}
+{% if brands_by_organisation %}
+<h2>
+    How to review new email branding
+</h2>
+<p>
+    Make sure new email branding can remain in the branding pool. Check:
+</p>
+<ul>
+    <li>if it’s a duplicate</li>
+    <li>image quality</li>
+    <li>if it can be used by more than one service</li>
+    <li>branding name</li>
+</ul>
+
+<h3>1. Duplicates</h3>
+<p>
+    If the same logo is in the branding pool twice then keep whichever one looks best.
+</p>
+
+<h3>2. Image quality</h3>
+
+<p>Check the image has a transparent background and is:</p>
+<ul>
+    <li>landscape</li>
+    <li>not distorted or stretched</li>
+    <li>not low resolution</li>
+</ul>
+
+<h4>Text in a logo</h4>
+<p>
+    Some logos may contain text. If the logo contains the name of the service or organisation, it must be clear to read. Ideally, text size ideally should not be less than 14px.
```

Comment:
"Ideally, text size ideally" probably can drop one of those

Reply:
```suggestion
    Some logos may contain text. If the logo contains the name of the service or organisation, it must be clear to read. Ideally, text size should be 14px or larger.
```